File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    answer = 0
    arr = s.split()
    
    for i in range(0, len(arr)):
        if arr[i] != 'Z':
            answer += int(arr[i])
        else:
            answer -= int(arr[i-1])
        
        print(answer)
        
    return answer

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.swift
import Foundation

func solution(_ s:String) -> Int {
    var answer:Int = 0
    var arr = s.split(separator: " ")
    
    for i in 0..<arr.count {
        if arr[i] != "Z" {
            answer += Int(arr[i])!
        } else {
            answer -= Int(arr[i-1])!
        }
    }
        
    return answer
}